src/cogent3/maths/stats/distribution.py

```diff
@@ -5,7 +5,7 @@
 
 from numpy import arctan as atan
 from numpy import array, exp, sqrt
-from scipy.stats import f, norm, t
+from scipy.stats import binom, f, norm, t
```

Comment:
delete the new import

Reply:
Now I revert the change within the function to the original and delete the new imports. Please have a look : )